Rename action into update (#90)
meilisearch/documentation: advanced_guides/asynchronous_updates.md

MeiliSearch is an **asynchronous API**. It means that the API does not behave as you would typically expect when handling the request's responses.

> Some updates are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the update.

Reviewer: ```suggestion
Some processes are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains an update identifier to track the progress of the process.
```

### Async flow

> - When making a write request (*create/update/delete*) against the search engine, it stores the writing udpate received in a queue and returns an `updateId`. With this id, the specific update execution is trackable.

Reviewer: ```suggestion
- When making a write request (*create/update/delete*) against the search engine, it stores the writing process received in a queue and returns an `updateId`. With this id, the specific process update is trackable.
```

  M->>-Q: dequeue update 2
</mermaid>

> ### Which updates are async?

Reviewer: ```suggestion
### Which actions are async?
```
or which `process, operation`.  `Which update` is not correct in context.

Reviewer 2: I prefer `operation`.

Reviewer: Would you prefer operation on the above change requests too ?

Reviewer 2: Yes, it seems better.

> Every update which could be compute-expensive is asynchronous. These include:

Reviewer: same as above: operation, actions or process

- Create/update a schema
- Update index settings
- Add/update/delete documents

### Understanding updates

Updates returns the following information:

> * **status**: State of the update (enqueued, processed)

Reviewer: same as above: operation, actions or process

> * **status**: State of the update (enqueued, processed)

Reviewer: ```suggestion
* **status**: State of the operation (enqueued, processed)
```

> * **status**: State of the update (enqueued, processed)

Reviewer 2: ```suggestion
* **status**: The state of the update (enqueued, processed, or failed).
```

> * **updateId**: Id of the update

Reviewer 2: ```suggestion
* **updateId**: The id of the update.
```

> * **type**: Information about the update type

Reviewer: same as above: operation, actions or process

> * **type**: Information about the update type

Reviewer: ```suggestion
* **type**: Information about the operation type
```

> * **type**: Information about the update type

Reviewer 2: ```suggestion
* **type**: An information about type of the update.
```

> * **enqueuedAt**: Date at which the update has been added to the queue

Reviewer: same as above: operation, actions or process

> * **enqueuedAt**: Date at which the update has been added to the queue

Reviewer: ```suggestion
* **enqueuedAt**: Date at which the operation has been added to the queue
```

> * **enqueuedAt**: Date at which the update has been added to the queue

Reviewer 2: ```suggestion
* **enqueuedAt**: The date at which the update has been added to the queue.
```

> * **processedAt**: Date ate which the update has done processing.

Reviewer: same as above: operation, actions or process

Reviewer 2: ```suggestion
* **processedAt**: Date at which the operation has done processing.
```

Reviewer: This suggestion is good

> * **processedAt**: Date ate which the update has done processing.

Reviewer 2: ```suggestion
* **processedAt**: The date at which the operation has been processed.
```
